auctions/english-auction/packages/scripts/src/helpers/createConfig.ts

```diff
@@ -0,0 +1,5 @@
+import type { Config } from '../types';
+
+export function createConfig(config: Config) {
+  return config;
+}
```

Comment:
I don't even think the file is used.  I'll delete it.

Reply:
ok it is actually used in the `english-auction.config.ts` file.  I tried messing around and just exporting the config object, but it's not working.  I don't have a ton of time to mess around with this so it may be something to come back to.  I'll ask the frontend team for some clarification around the configuration setup.